### ISSUE TITLE
Minified UMD bundle

### ISSUE THREAD

**Issue by FredyC** [mobxjs/mobx-react-lite#3]
One leftover from my work. For some reason, the uglify plugin is failing to parse source files properly. I am not sure why is it happening.

```js
$ node build-rollup.js
  20 |     }
  21 |     function configureDOM() {
> 22 |         const { unstable_batchedUpdates } = require("react-dom");
     |        ^ Unexpected token: keyword (const)
  23 |         mobx.configure({ reactionScheduler: unstable_batchedUpdates });
  24 |     }
  25 |     function configureNative() {
{ SyntaxError: Unexpected token: keyword (const)
    at JS_Parse_Error.get (eval at <anonymous> (D:\workspace\github\mobx-react\node_modules\uglify-js\tools\node.js:20:1), <anonymous>:71:23)
```

**Comment by mcjazzyfunky** [comment]
AFAIK, ugilfyjs does not support ES2015 and higher.
I think maybe you should consider to use ```"target": "ES5"``` in tsconfig.json or if you really want a ES201x module then maybe "rollup-plugin-uglify-es" will do the job.

**Comment by FredyC** [comment]
@mcjazzyfunky You are right, target ES5 did the trick, but that's not likely what we want here. Times of ES5 is so over in my opinion :) Are you willing to experiment with the plugin you have mentioned?

I've also heard there is some alternative to uglify built from fresh but with the same API, but I just cannot recall the name of that project.

**Comment by mcjazzyfunky** [comment]
Ooops, rollup-plugin-ugilfy-es seems to be deprecated:
As written in its README.md

> DEPRECATED, see rollup-plugin-terser instead

@FredyC was "terser" that thing you were talking about by any chance?

**Comment by FredyC** [comment]
@mcjazzyfunky Yea, Terses, thank you! :) Would be lovely if you can send in PR :)

**Comment by FredyC** [comment]
Solved in #5 Thanks!
